### DISCUSSION TITLE
Mask ignored?

### DISCUSSION THREAD

**Discussion by thomasfals** [blakeblackshear/frigate#15026] [Ask A Question] [answered]
Hi

I have a mask to eliminate person detection in a certain area. However, I still get triggers for person in this area. Frigate has been restarted - what am I missing?

![Screenshot 2024-11-16 at 23 19 46](https://github.com/user-attachments/assets/54450f72-c0ea-4ab9-9673-5f8991fcd5fa)
![Screenshot 2024-11-16 at 23 11 12](https://github.com/user-attachments/assets/53953007-18df-4962-914e-1b728aebea13)

**Comment by hawkeye217** [comment, marked as answer]
Motion masks don't prevent objects from being detected. See [this tip](https://github.com/blakeblackshear/frigate/discussions/13040).

If you are using an object mask, that should cover the bottom center of the bounding box. See [the docs](https://docs.frigate.video/configuration/masks/#object-filter-masks).
